### PR TITLE
Page titles component

### PR DESCRIPTION
**Pull request by alejobh**
## Summary

The purpose of this branch is to add the Component pageTitle, check the screenshot.

## Screenshots

image

## Known isssues/Notes

The first branch pushed was called ‘proyect-setup‘ Link: https://github.com/wolox-training/abh-react/tree/proyect-setup didn't contain the assets folder so i included it in this new branch and i also updated the .gitignore to ignore the actual files that needed to be ignored and removed from the branch

## Trello Card

https://trello.com/c/1nLnKBH3

### CODE REVIEW

File: maquetado/scss/application.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import 'titles';
+@import 'variables';
+
+body {
+	font-family: $font-family;
```
Comment (henryzarza) [inline maquetado/scss/application.scss]: The indentation it's must be 2 spaces

File: maquetado/scss/titles.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+@import 'variables';
+
+.title {
+  align-items: center;
+  color: $black;
+  display: flex;
+  font-size: $text-big;
+  padding-left: 51px;
+  position: relative;
+  height: $title-decoration-height;
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: Remember the properties must be order by alphabetically

File: maquetado/scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  &::before {
+    background: (url(assets/triangle.svg));
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: It's not necessary wrapper `(url(assets/triangle.svg))` you just put `url(assets/triangle.svg)` and below you have the property `background-repeat` so I consider that you should change the currently property `background` for `background-image`

File: maquetado/scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    background-repeat: no-repeat;
+    content: '';
+    height: $title-decoration-height;
+    left: 0;
+    position: absolute;
+    width:  $title-decoration-width;
+    z-index: -1;
+  }
+
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: Delete this line empty

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Comment by braiandickson** [comment]
@alejobh, resolve conflicts updating your branch with the last merge 👍

**Review by henryzarza — APPROVED** [review]

**Pull request merged**